src/index.js

```diff
@@ -22,7 +22,14 @@ const is = require('is');
 const through = require('through2');
 const util = require('util');
 
-const libVersion = require('../../package.json').version;
+let libVersion;
```

Comment:
Eugh, this is less than idea.  Is this because of the use of `ts-node`?  In most other places, we just do the import to the relative path assuming it's running from `build/src`.

Reply:
@ofrobots for thoughts here too

Reply:
Yeah, this is pretty bad, but I don't know a good way around it. I even had to flip the order to get 100% test coverage back...

There are two problems here:
- `ts-node` skips the build step, which makes it super easy to work with (it's a huge productivity boost), but the source needs to find the package.json (unless I set it to an unknown value during development?).
- I can't copy the package.json into the `build/` folder since NPM will then recursively apply these settings. `npm pack`, for example, will then not copy the right files and try to include "build/build/src" instead.

I hope there are better ways to solve these two problems!

Reply:
In https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs we had to deal with a similar issue where some files (compiled protos) need to be able to resolvable relative to both from compiled and uncompiled sources. To deal with this we added an extra top directory (`/ts`) so that the number of `..`'s needed to get to `package.json` is constant.

Another alternative would be to use something like [pkg-up](https://github.com/sindresorhus/pkg-up) and write the code as:

```ts
const pkgUp = require('pkg-up');
const libVersion = require(await pkgUp());  // top-level await, where art thou?
```

WDYT?

Reply:
I do like the work-around that you described by adding another level of folders, but it would be a huge pain to merge my pending PRs. So instead I went with pkg-up (which also offers a synchronous API). The loading of the package version is now much cleaner!